library/core/src/ptr/mod.rs

```diff
@@ -65,6 +65,12 @@
 //! separate allocated object), heap allocations (each allocation created by the global allocator is
 //! a separate allocated object), and `static` variables.
 //!
+//! It is guaranteed that an allocated object never spans more than `isize::MAX` bytes.
```

Comment:
Even without agreeing on 0 == null, you can say that an allocated object can never contain NULL, i.e., the pointer returned by `ptr::null`, right?

It seems really odd to mention that it can't contain MAX but *not* mention that it can't contain NULL.

Reply:
Good point; how does that look?